composer.json

```diff
@@ -29,59 +29,59 @@
     "composer/installers": "^1.11",
     "mautic/core-composer-scaffold": "4.x-dev",
     "mautic/core-project-message": "4.x-dev",
-    "mautic/core-lib": "6.x-dev",
-    "mautic/grapes-js-builder-bundle": "6.x-dev",
-    "mautic/plugin-clearbit": "6.x-dev",
-    "mautic/plugin-cloudstorage": "6.x-dev",
-    "mautic/plugin-crm": "6.x-dev",
-    "mautic/plugin-emailmarketing": "6.x-dev",
-    "mautic/plugin-focus": "6.x-dev",
-    "mautic/plugin-fullcontact": "6.x-dev",
-    "mautic/plugin-gmail": "6.x-dev",
-    "mautic/plugin-outlook": "6.x-dev",
-    "mautic/plugin-social": "6.x-dev",
-    "mautic/plugin-tagmanager": "6.x-dev",
-    "mautic/plugin-zapier": "6.x-dev",
-    "mautic/theme-1-2-1-2-column": "6.x-dev",
-    "mautic/theme-1-2-1-column": "6.x-dev",
-    "mautic/theme-1-2-column": "6.x-dev",
-    "mautic/theme-1-3-1-3-column": "6.x-dev",
-    "mautic/theme-1-3-column": "6.x-dev",
-    "mautic/theme-attract": "6.x-dev",
-    "mautic/theme-aurora": "6.x-dev",
-    "mautic/theme-blank": "6.x-dev",
-    "mautic/theme-blend": "6.x-dev",
-    "mautic/theme-brienz": "6.x-dev",
-    "mautic/theme-capture": "6.x-dev",
-    "mautic/theme-cards": "6.x-dev",
-    "mautic/theme-chord": "6.x-dev",
-    "mautic/theme-confirmme": "6.x-dev",
-    "mautic/theme-connect-through-content": "6.x-dev",
-    "mautic/theme-creative": "6.x-dev",
-    "mautic/theme-eclipse": "6.x-dev",
-    "mautic/theme-educate": "6.x-dev",
-    "mautic/theme-formscape": "6.x-dev",
-    "mautic/theme-fresh-center": "6.x-dev",
-    "mautic/theme-fresh-fixed": "6.x-dev",
-    "mautic/theme-fresh-left": "6.x-dev",
-    "mautic/theme-fresh-wide": "6.x-dev",
-    "mautic/theme-gallery": "6.x-dev",
-    "mautic/theme-goldstar": "6.x-dev",
-    "mautic/theme-make-announcement": "6.x-dev",
-    "mautic/theme-mono": "6.x-dev",
-    "mautic/theme-neopolitan": "6.x-dev",
-    "mautic/theme-oxygen": "6.x-dev",
-    "mautic/theme-paprika": "6.x-dev",
-    "mautic/theme-reachout": "6.x-dev",
-    "mautic/theme-showcase": "6.x-dev",
-    "mautic/theme-simple-text": "6.x-dev",
-    "mautic/theme-skyline": "6.x-dev",
-    "mautic/theme-sparse": "6.x-dev",
-    "mautic/theme-sunday": "6.x-dev",
-    "mautic/theme-survey": "6.x-dev",
-    "mautic/theme-trulypersonal": "6.x-dev",
-    "mautic/theme-vibrant": "6.x-dev",
-    "mautic/theme-welcome": "6.x-dev"
+    "mautic/core-lib": "^6.0.3",
```

Comment:
Comparing it with the [6.0.2 release](https://github.com/mautic/recommended-project/pull/118/files) there was no `^`. Let's remove it to avoid unexpected behavior.

Reply:
done